### PR TITLE
RepGraph Test Actor when there is no PC

### PR DESCRIPTION
**Pull request by improbable-danny**
#### Description
A RepGraph test actor which is always replicated to Spatial 

#### Primary reviewers
@m-samiec

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase.h [hunk 1]
```diff
@@ -10,7 +10,7 @@
  * A replicated Actor with a Cube Mesh, used as a base for Actors used in spatial tests.
  */
 UCLASS()
-class AReplicatedTestActorBase : public AActor
+class SPATIALGDKFUNCTIONALTESTS_API AReplicatedTestActorBase : public AActor
```
Comment (m-samiec) [inline SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase.h:13]: Do we need to expose this?

File: SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase_RepGraph.h [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#include "CoreMinimal.h"
+#include "GameFramework/Actor.h"
+#include "ReplicatedTestActorBase.h"
+#include "ReplicatedTestActorBase_RepGraph.generated.h"
+
+/**
+ * A replicated Actor which is always replicated by RepGraph regardless of client view.
+ */
+UCLASS()
+class SPATIALGDKFUNCTIONALTESTS_API AReplicatedTestActorBase_RepGraph : public AReplicatedTestActorBase
```
Comment (m-samiec) [inline SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase_RepGraph.h:14]: You're gonna hate me, but maybe this actor should be named something else.
AReplicatedTestActorBase_RepGraphAlwaysReplicate ?

**Review by m-samiec — APPROVED** [review]

File: SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase_RepGraphAlwaysReplicate.cpp [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```
Comment (martin-improbable) [inline SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase_RepGraphAlwaysReplicate.cpp:1]: File is not necessary?

File: SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase_RepGraphAlwaysReplicate.h [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+/**
+ * A replicated Actor which is always replicated by RepGraph regardless of client view.
+ */
+UCLASS()
+class AReplicatedTestActorBase_RepGraphAlwaysReplicate : public AReplicatedTestActorBase
+{
+	GENERATED_BODY()
+
+public:
+	AReplicatedTestActorBase_RepGraphAlwaysReplicate();
```
Comment (martin-improbable) [inline SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase_RepGraphAlwaysReplicate.h:19]: Not necessary?

File: SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase_RepGraphAlwaysReplicate.h [hunk 5]
```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Actor.h"
```
Comment (martin-improbable) [inline SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase_RepGraphAlwaysReplicate.h:6]: not necessary?

**Review by razvan-at-improbable — APPROVED** [review]
LGTM.

**Pull request merged**